src/Fixer/Comment/HeaderCommentFixer.php

```diff
@@ -427,11 +427,7 @@ private function removeHeader(Tokens $tokens, int $index): void
 
             $content = Preg::replace('/\R?\h*$/', '', $content);
 
-            if ('' === $content) {
-                $tokens->clearAt($prevIndex);
```

Comment:
don't we need this to remove headers when the new header is configured as being `''` ?